test/connectors/GuidanceConnectorSpec.scala

```diff
@@ -18,31 +18,42 @@ package connectors
 
 import base.BaseSpec
 import uk.gov.hmrc.http.HeaderCarrier
-
+import mocks.{MockAppConfig, MockHttpClient}
 import scala.concurrent.ExecutionContext.Implicits.global
-
-class GuidanceConnectorSpec extends BaseSpec {
+import play.api.test.{DefaultAwaitTimeout, FutureAwaits}
+import play.api.libs.json.Json
+import models.ocelot.Process
+import scala.concurrent.Future
+
+class GuidanceConnectorSpec extends BaseSpec with MockHttpClient {
+
+  private trait Test extends MockHttpClient with FutureAwaits with DefaultAwaitTimeout {
+    val process: Process = Json.parse(models.ocelot.PrototypeJson.json).as[Process]
+    implicit val hc: HeaderCarrier = HeaderCarrier()
+    val gc: GuidanceConnector = new GuidanceConnector(mockHttpClient, MockAppConfig)
+    val endPoint: String = MockAppConfig.externalGuidanceBaseUrl + "/external-guidance/scratch/"
+  }
 
   "Calling the getProcess with an existing process ID" should {
-    "return a model representing the Ocelot Process" in {
-      val hc: HeaderCarrier = HeaderCarrier()
-      val target = new GuidanceConnector()
-      val expectProcess = Some(target.stubbedProcess)
-      val result = target.getProcess("ext90002")(hc, implicitly)
+    "return a model representing the Ocelot Process" in  new Test {
+      val result = gc.getProcess("ext90002")(hc, implicitly)
 
-      whenReady(result) { _ mustBe expectProcess }
+      whenReady(result) { _ mustBe Some(process) }
     }
   }
 
   "Calling the scratchProcess with an existing scratch process UUID" should {
 
-    "return a model representing the Ocelot Scatch Process" in {
-      val hc: HeaderCarrier = HeaderCarrier()
-      val target = new GuidanceConnector()
-      val expectProcess = Some(target.scratchStubbedProcess)
-      val result = target.scratchProcess("683d9aa0-2a0e-4e28-9ac8-65ce453d2730")(hc, implicitly)
+    "return a model representing the Ocelot Scatch Process" in new Test {
```

Comment:
The connector handles both successful and unsuccessful calls so a test for the later would be good.